apps/foundation-demo/src/app/checkout/actions/shipping.actions.ts

```diff
@@ -1,12 +1,12 @@
 import { Action } from '@ngrx/store';
 
 export enum ShippingActionTypes {
-  ToggleShippingFormAction = "[Foundation-Shipping] Toggle Shipping Form Action",
+  ToggleShowShippingFormAction = "[Foundation-Shipping] Toggle Show Shipping Form Action",
```

Comment:
This seems like it's outside the scope of this PR. Is it possible to cherry pick this commit into another PR?

Reply:
moved to WIP branch: https://github.com/graycoreio/daffodil/pull/39